### PR TITLE
Feature header

### PR DESCRIPTION
**Pull request by alanluqman**
In this pull request, we have created a header.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+function App() {
+  return (
+    <>
+      <BrowserRouter>
+        <Navigation />
+        <Routes>
+          <Route />
+          <Route />
+          <Route />
```
Comment (ogagaoghene) [inline src/App.js:13]: Route links no specified yet

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+* {
+  box-sizing: border-box;
```
Comment (ogagaoghene) [inline src/App.css:2]: set margin to 0 
set padding to 0

File: src/component/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <h1>Space Travelers Hub</h1>
+          </div>
+        </Link>
+        <div>
+          <NavLink className="nav-links" style={activeStyle} to="/">Rockets</NavLink>
+          <NavLink className="nav-links" style={activeStyle} to="/missions">Missions</NavLink>
+          <NavLink className="nav-links my-profile" style={activeStyle} to="/myprofile">My Profile</NavLink>
+        </div>
+      </div>
+    </div>
```
Comment (ogagaoghene) [inline src/component/Header.jsx:27]: I would love you use html semantic tags like header tager and nav tag here

**Review by ogagaoghene — COMMENTED** [review]
Good job far!! 👍 Please make those changes.

**Comment by alanluqman** [comment]
Thanks for your review, I changed all requirements

**Review by ogagaoghene — APPROVED** [review]

**Pull request merged**